litmus-portal/graphql-server/graph/usermanagement.graphqls

```diff
@@ -14,11 +14,13 @@ type User {
 }
 
 input CreateUserInput {
+  user_id: String!
   username: String!
   email: String
   company_name: String
   name: String
   project_name: String!
+  role: String!
```

Comment:
This role is not needed in the project db, the role is there in the user claims of jwt just reuse that